internal/controller/components/modelregistry/modelregistry_support.go

```diff
@@ -29,6 +29,7 @@ var (
 		"IMAGES_GRPC_SERVICE":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
 		"IMAGES_REST_SERVICE":           "RELATED_IMAGE_ODH_MODEL_REGISTRY_IMAGE",
 		"IMAGES_OAUTH_PROXY":            "RELATED_IMAGE_OSE_OAUTH_PROXY_IMAGE",
+		"IMAGES_CATALOG_DATA":           "RELATED_IMAGE_ODH_MODEL_CATALOG_DATA_IMAGE",
```

Comment:
this 
/lgtm

but it's in a PR not yet merged:
- https://github.com/opendatahub-io/model-registry-operator/pull/299/files#diff-54fa1a6b6f0417fb589ec16b1b67064140cb5de547f52111e7bbad93045af97aR5

Reply:
This PR will set the env variable in operator deployment, so it won't hurt the operator if the env variable doesn't exist or is not being used yet. 
But, both PRs will need to be merged for this to work.